public/app/features/folders/state/actions.ts

```diff
@@ -122,9 +121,8 @@ export function addFolderPermission(newItem: NewDashboardAclItem): ThunkResult<v
 
 export function createNewFolder(folderName: string): ThunkResult<void> {
   return async dispatch => {
-    // @ts-ignore
-    const newFolder = await getBackendSrv().createFolder({ title: folderName });
+    const newFolder = await getBackendSrv().post('/api/folders', { title: folderName });
     appEvents.emit(AppEvents.alertSuccess, ['Folder Created', 'OK']);
-    dispatch(updateLocation({ path: newFolder.url }));
+    dispatch(updateLocation({ path: locationUtil.stripBaseFromUrl(newFolder.url) }));
```

Comment:
ah, so the API call returns URL with the base url applied?

Reply:
correct!